specs/pvs/corla.pvs

```diff
@@ -6,7 +6,18 @@
 %%%
```

Comment:
At line 866, I would change this comment for `  margin_overstatement: TYPE = nat`:
```
% @description A circumstance in which the audit board’s
% interpretation of ballot markings reveals that the winner with the
% fewest votes received fewer votes than the voting system’s
% interpretation of the same markings as reflected in the CVR.

% @example For example, if the CVR reflects an undervote in the
% audited contest, and the audit board’s interpretation of the
% corresponding paper ballot reflects a vote for the loser with the
% most votes in that contest, the CVR contains a one-vote
% overstatement. if the CVR reflects a vote for the winner with the
% fewest votes, and the audit board’s interpretation of the paper
% ballot reflects a vote for the loser with the most votes, the
% preliminary results contain a two-vote overstatement.
  margin_overstatement: TYPE = nat
```
to....

```
% @description A circumstance in which the audit board’s
% interpretation of ballot markings reveals that the winner with the
% fewest votes had a smaller margin than in the voting system’s
% interpretation of the same markings as reflected in the CVR.
```
and capitalize the "if" that starts the second example in the next paragraph.

A subtle difference, but just to be clear.

It needs to change in a similar way for the understatement definition.